cmd/node/config/systemSmartContractsConfig.toml

```diff
@@ -11,6 +11,8 @@
     MaxNumberOfNodesForStake = 36
     UnJailValue = "2500000000000000000" #0.1% of genesis node price
     ActivateBLSPubKeyMessageVerification = false
+    StakeLimitPercentage = 1.0
```

Comment:
Do we need also stake limit percentage? Why not just nodeLimitPercentage? Is this 1 percent or 100 percent?

Reply:
This is 1 percent.

Reply:
changed to 0.01 as in ecomonics.toml

Reply:
ok!